<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1011</width>
      <height>981</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>481</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Segoe Print&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600; color:#00007f;&quot;&gt;Welcome to Fifteen Puzzle Game&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>590</y>
        <width>511</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 127);
font: 8pt &quot;Segoe Print&quot;;
</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#00007f;&quot;&gt;Realised by EL IDRISSI Rkia &amp;amp; BOUNOUA Oum Koulthoume&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>350</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 0);
font: 22pt &quot;Gill Sans MT Condensed&quot;;</string>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>460</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 67, 30);
font: 22pt &quot;Gill Sans MT Condensed&quot;;</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;How to play&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 127);
font: 22pt &quot;Gill Sans MT Condensed&quot;;
</string>
      </property>
      <property name="text">
       <string>How to play</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>70</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/images/icons8-puzzle-pieces-64.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/images/icons8-puzzle-pieces-64.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>200</y>
        <width>311</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/images/eidia.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="BoardView" name="board_view" native="true">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>641</width>
        <height>381</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 127);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>590</y>
        <width>141</width>
        <height>42</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 22pt &quot;Tw Cen MT Condensed&quot;;
background-color: rgb(255, 255, 127);</string>
      </property>
      <property name="text">
       <string>Back to home</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>510</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/images/icons8-artificial-intelligence-64.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="new_game_btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Gill Sans MT Condensed</family>
        <pointsize>22</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;scramble          &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 127);
font: 22pt &quot;Gill Sans MT Condensed&quot;;
</string>
      </property>
      <property name="text">
       <string>scramble</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/icons/img/new.png</normaloff>:/icons/img/new.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="start_again_btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Gill Sans MT Condensed</family>
        <pointsize>22</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/icons8-restart-96.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 22pt &quot;Gill Sans MT Condensed&quot;;
background-color: rgb(255, 85, 51);</string>
      </property>
      <property name="text">
       <string>restart</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/icons/img/replay.png</normaloff>:/icons/img/replay.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="back_move_btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>320</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Tw Cen MT Condensed</family>
        <pointsize>22</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 22pt &quot;Tw Cen MT Condensed&quot;;
background-color: rgb(255, 255, 0);</string>
      </property>
      <property name="text">
       <string>back</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/icons/img/back.png</normaloff>:/icons/img/back.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>530</y>
        <width>121</width>
        <height>36</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/icons8-artificial-intelligence-64.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 22pt &quot;Gill Sans MT Condensed&quot;;</string>
      </property>
      <property name="text">
       <string>Complexity</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>540</y>
        <width>102</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="complexity_num_lbl">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="complexity_num_sldr">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <zorder>new_game_btn</zorder>
     <zorder>board_view</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>pushButton_4</zorder>
     <zorder>complexity_num_lbl</zorder>
     <zorder>label_6</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BoardView</class>
   <extends>QWidget</extends>
   <header>boardview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>complexity_num_sldr</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>complexity_num_lbl</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>486</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>complexity_num_sldr</sender>
   <signal>valueChanged(int)</signal>
   <receiver>complexity_num_lbl</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
